ui/src/app/pages/access/access-renewal-page.tsx

```diff
@@ -0,0 +1,188 @@
+        <div>Note that you are obliged by the Terms of Use of the Workbench to provide keep your profile
+          information up-to-date at all times.
+        </div>
+        {
+          isComplete(getExpirationTimeFor('profileConfirmation'))
+            ? <CompletedButton buttonText='Confirmed' wasBypassed={false}/>
+            : <Button style={{marginTop: 'auto', height: '1.6rem', width: '4.5rem'}}>Review</Button>
+        }
+      </RenewalCard>
+      {/* Publications */}
+      <RenewalCard step={2}
+                   TitleComponent={() => 'Report any publications or presentations based on your research using the Researcher Workbench'}
+                   lastCompletion={publicationsLastConfirmedTime}
+                   nextReview={getExpirationTimeFor('publicationConfirmation')}>
+        <div>The <AoU/> Publication and Presentation Policy requires that you report any upcoming publication or
+             presentation resulting from the use of <AoU/> Research Program Data at least two weeks before the date of publication.
+             If you are lead on or part of a publication or presentation that hasn’t been reported to the
+             program, <a target='_blank'  href={'https://redcap.pmi-ops.org/surveys/?s=MKYL8MRD4N'}>please report it now.</a>
+        </div>
+        <div style={{marginTop: 'auto', display: 'grid', columnGap: '0.25rem', gridTemplateColumns: 'auto 1rem 1fr', alignItems: 'center'}}>
+          {
+            isComplete(getExpirationTimeFor('publicationConfirmation'))
+              ? <CompletedButton style={{gridRow: '1 / span 2'}} buttonText='Confirmed' wasBypassed={false}/>
+              : <Button disabled={publications === null}
+                    style={{gridRow: '1 / span 2', height: '1.6rem', width: '4.5rem', marginRight: '0.25rem'}}>Confirm</Button>
+          }
+          <RadioButton id={noReportId} style={{justifySelf: 'end'}} checked={publications === true} onChange={() => setPublications(true)}/>
+          <label htmlFor={noReportId}> At this time, I have nothing to report </label>
+          <RadioButton id={reportId} style={{justifySelf: 'end'}} checked={publications === false} onChange={() => setPublications(false)}/>
+          <label htmlFor={reportId}>Report submitted</label>
+        </div>
+      </RenewalCard>
+      {/* Code of Conduct */}
+      <RenewalCard step={3}
+                   TitleComponent={() => <div><AoU/> Responsible Conduct of Research Training</div>}
+                   lastCompletion={complianceTrainingCompletionTime}
+                   nextReview={getExpirationTimeFor('complianceTraining')}>
+        <div> You are required to complete the refreshed ethics training courses to understand the privacy safeguards and
+          the compliance requirements for using the <AoU/> Dataset.
+        </div>
+        {
+          !!complianceTrainingBypassTime || isComplete(getExpirationTimeFor('complianceTraining'))
```

Comment:
nit: `!!` unnecessary here since it's already coerced in a boolean context (ternary predicate)

here and L181